### PR TITLE
EN_Reboot: Hotfixes

### PR DESCRIPTION
**Pull request by Frey8914**
-Fix Mining Rig Defense.
-Fix UI.

### CODE REVIEW

File: Translated/Tutorials/tut_006414.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
 Title#0,"""Collect Signs"""
-text0001#0,"""During battle, special Photon crystals called <yellow>\""Collect Signs\""<c> will appear in the field. When collected, they award <yellow>\""Merit Points\""<c>, which you can use to access support functions of Mining Rigs and Delivery Devices.<br><br><yellow>※Merit Points are awarded to all players at once, and each player can<br>\u3000use them on their own choice of support functions.<c>"""
+text0001#0,"""During battle, special Photon crystals called <yellow>\""Collect Signs\""<c> will appear in the field. When collected, they award <yellow>\""Obtained Points\""<c>, which you can use to access support functions of Mining Rigs and Delivery Devices.<br><br><yellow>※Obtained Points are awarded to all players at once, and each player can<br>\u3000use them on their own choice of support functions.<c>"""
```
Comment (SynthSy) [inline Translated/Tutorials/tut_006414.csv:2]: no

File: Translated/Tutorials/tut_006419.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
-text0003#0,"""If you collect enough Collect Signs and access a Mining Rig, you will be able to use the <yellow>\""Durability Recovery\""<c> and <yellow>\""Barrier Shield\""<c> support functions.<br><br><yellow>\""Mobile Cannons\""<c> are powerful special weapons that can be summoned from Delivery Devices. While equipped with one, you cannot access your normal weapons and gain a special dedicated moveset instead."""
+text0002#0,"""During battle, special Photon crystals called <yellow>\""Collect Signs\""<c> will appear in the field. When collected, they award <yellow>\""Obtained Points\""<c>, which you can use to access support functions of Mining Rigs and Delivery Devices.<br><br><yellow>※Obtained Points are awarded to all players at once, and each player can<br>\u3000use them on their own choice of support functions.<c>"""
```
Comment (SynthSy) [inline Translated/Tutorials/tut_006419.csv:3]: no

File: Translated/UI/ui_mainmode_reb.csv [hunk 3]
```diff
@@ -441,7 +441,7 @@ TargetInfo_Name#0,"""$(0) $(1)"""
 TargetInfo_WeekAndResist#0,"""$(0)"""
 TargetInfo_WeakPoint#0,"""Weakness"""
 TargetInfo_Resist#0,"""Resistance"""
-Title#0,"""Simple Status"""
+Title#0,"""Basic Status"""
```
Comment (SynthSy) [inline Translated/UI/ui_mainmode_reb.csv:444]: no

File: Translated/UI/ui_quest_reb.csv [hunk 4]
```diff
@@ -16,7 +16,7 @@ E_eario_Dolls_1120001_Open#0,"""Notifying all ARKS: We are beginning a mission t
 E_defense_Common_End#0,"""All DOLLS have been annihilated, and the defense target's safety is confirmed. Thank you for your cooperation in this mission."""
 E_eario_defense_1120003_Notice#0,"""Emergency Alert: A large number of DOLLS have appeared in the Aerio region. Defense preparations are now underway. All ARKS, please prepare to sortie. <yellow>(For players with a Power Level of $(0) or higher)<c>"""
 E_eario_defense_1120003_Open#0,"""Notifying all ARKS: We are now beginning operations to defend ourselves against the large number of DOLLS that have appeared in the Aerio region. <yellow>(For players with a Power Level of $(0) or higher)<c>"""
-Title_Quest_Info_1#0,"""Simple Quest Information"""
+Title_Quest_Info_1#0,"""Basic Quest Information"""
```
Comment (SynthSy) [inline Translated/UI/ui_quest_reb.csv:19]: no

File: Translated/UI/ui_mainmode.csv [hunk 5]
```diff
@@ -6040,7 +6040,7 @@ Menu_Open_Win_Menu03#0,"""R Ring Shop"""
 Menu_Open_Win_Menu04#0,"""Combined Ring Shop"""
 Menu_Open_Win_Close#0,"""Return"""
 Menu_Open_Disable#0,"""The Gathering Terminal cannot be used now.<br><br>You will be able to use it if you complete<br>Sophia's Client Orders in Franca's Cafe."""
-Title_Gat_Simple_Status#0,"""Basic Gathering Details"""
+Title_Gat_Simple_Status#0,"""Basic Gathering Information"""
```
Comment (SynthSy) [inline Translated/UI/ui_mainmode.csv:6043]: no

File: Translated/UI/config.csv [hunk 6]
```diff
@@ -568,8 +568,8 @@ Config_011_Choices_3#0,"""Prem. Storage"""
 Config_011_Choices_4#0,"""OTP Storage"""
 Config_011_Choices_5#0,"""PSO2es Storage"""
 Config_011_Hint#0,"""Set the initial cursor position when the Storage UI is displayed. Applies to opening Storage, to putting items up for sale in your My Shop and to buying and selling items at NPC shops."""
-Config_012_Name#0,"""Simple Gathering Details"""
-Config_012_Hint#0,"""Set whether to display simple Gathering information on the Main Menu."""
+Config_012_Name#0,"""Basic Gathering Information"""
```
Comment (SynthSy) [inline Translated/UI/config.csv:571]: no

**Review by SynthSy — CHANGES_REQUESTED** [review]
Some things on here I see are fine, others I will assume it's to satisfy your OCD. This is a won't fix issue.

File: Translated/Tutorials/tut_006414.csv (hunk 1 above)
Comment (Frey8914) [reply Translated/Tutorials/tut_006414.csv:2]: They're not called Merit Points.

Comment (SynthSy) [reply Translated/Tutorials/tut_006414.csv:2]: 取得 is `to obtain` `to acquire`, thus merit would work in this case.

File: Translated/UI/ui_mainmode.csv (hunk 5 above)
Comment (Frey8914) [reply Translated/UI/ui_mainmode.csv:6043]: The column above has read Basic Mission Information in classic.
This is a long overdue fix.
簡易ミッション情報 Basic Mission Information
簡易ギャザリング情報 Basic Gathering Details

Comment (SynthSy) [reply Translated/UI/ui_mainmode.csv:6043]: We'd prefer it if you don't change it, due to spacing issues.

**Comment by SynthSy** [comment]
Overall, if you're unwilling to make the changes requested, we will close this PR.

**Pull request closed without merging**